fe/fe-core/src/main/java/com/starrocks/analysis/AddComputeNodeStmt.java

```diff
@@ -0,0 +1,72 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.analysis;
+
+import com.google.common.base.Strings;
+import com.starrocks.common.AnalysisException;
+import com.starrocks.common.Config;
+import com.starrocks.common.DdlException;
+import com.starrocks.common.ErrorCode;
+import com.starrocks.common.ErrorReport;
+import com.starrocks.common.Pair;
+import com.starrocks.common.UserException;
+import com.starrocks.qe.ConnectContext;
+import com.starrocks.server.GlobalStateMgr;
+import com.starrocks.sql.ast.AstVisitor;
+import com.starrocks.system.SystemInfoService;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AddComputeNodeStmt extends AlterSystemStmtNew {
```

Comment:
How about naming it as `AddNodeStmt` and make `compute` as one of node types.

Reply:
I don't think it needs to be changed to AddNodeStmt for the time being. This change to AddNodeStmt means that AddNodeStmt is the parent class of AddComputeNodeStmt, or it means that when AstBuilder parses the statement, compute/backend is passed as a type to the construction parameter? But I see that the new BE statement has been implemented. I feel that it is not meaningful to change it to AddNodeStmt, and there are only two kinds of nodes now, which is easy to cause misunderstanding.